README.md

```diff
@@ -18,6 +18,7 @@ Welcome to the central repository for GitHub Actions at the DfE!
 * [setup-cf-cli](setup-cf-cli)
 * [turnstyle](turnstyle)
 * [validate-key-vault-secrets](validate-key-vault-secrets)
+* [fetch-key-vault-secrets](fetch-key-vault-secrets)
```

Comment:
I think this list is alphabetical, I don't think it matters really but might be worth sticking with that.